README.md

```diff
@@ -8,7 +8,7 @@ Saga allows defining compensation functions for each step that will be automatic
 You can read more details about this pattern here https://microservices.io/patterns/data/saga.html#example-choreography-based-saga
 
 # Installing
-```go get github.com/itimofeev/go-saga```
```

Comment:
I think that we should keep previous import in order to merge this MR. It'll be strange, if repository will be itimofeev, but module name will contain another name.